connd/qconnectionagent.cpp

```diff
@@ -30,6 +30,10 @@
 #define CONND_PATH "/Connectiond"
 #define CONND_SESSION_PATH = "/ConnectionSession"
 
+Q_DECLARE_LOGGING_CATEGORY(connAgent)
```

Comment:
No need for this, the below Q_LOGGING_CATEGORY instantiates the logging. Separate declaration needed only if there's a header and logging used from multiple places.